dockstore-integration-testing/src/test/resources/dockstore.yml

```diff
@@ -17,7 +17,7 @@ googleClientSecret: <fill me in>
 googleRedirectURI: http://<fill me in>:8080/auth/tokens/google.com
 
 externalConfig: 
-  basePath: /
+  basePath: /api/
   hostname: localhost
   scheme: http
   port: 8080
```

Comment:
For line 57 below, what happens if this is changed to "validate" as per the override above?

Reply:
too many tests use this file and need an empty db (by using create).  Changing this to validate causes a bunch of issues.

Reply:
No tests should (prescriptive, not descriptive) be using hbm2ddl create. 
Please change this and use dropAndRecreateNoTestData to create the database, not hbm2ddl.
If the fixes drag on, feel free to spin that off into a PR against develop.